adapters/akcelo/akcelo.go

```diff
@@ -0,0 +1,187 @@
+package akcelo
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/buger/jsonparser"
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v3/adapters"
+	"github.com/prebid/prebid-server/v3/config"
+	"github.com/prebid/prebid-server/v3/errortypes"
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+	"github.com/prebid/prebid-server/v3/util/jsonutil"
+	"github.com/prebid/prebid-server/v3/util/ptrutil"
+	"net/http"
+	"net/url"
+)
+
+type adapter struct {
+	uri *url.URL
+}
+
+type extObj struct {
+	Akcelo json.RawMessage `json:"akcelo"`
+}
+
+var NoValidSiteIdError = &errortypes.BadInput{Message: "Cannot find valid siteId"}
+var NoValidImpError = &errortypes.BadInput{Message: "No valid Imp"}
```

Comment:
Great use of the Go pattern for errors. These do not need to be export, so please start the name with a lower case letter.